src/atoms/Divider/Divider.native.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import View from '../View';
+import { getColor, getColorKeys } from '../../_helpers/theme';
+
+const StyledView = styled(View)`
+  height: ${(props) => (props.dividerDirection === 'horizontal' ? '1px' : '100%')};
+  width: ${(props) => (props.dividerDirection === 'horizontal' ? '100%' : '1px')};
+  background-color: ${(props) => getColor(props.theme, props.color)};
+`;
+
+const Divider = ({ direction, color }) => <StyledView dividerDirection={direction} color={color} />;
```

Comment:
you don't need this. A styled component becomes a React compinent in the end. So it'll get all the props